guest-agent/VfioService/VfioService/Program.cs

```diff
@@ -18,10 +18,14 @@ static void Main()
             using (var manager = new ClientManager(form))
             {
                 form.ClientManager = manager;
+                InterceptMouse.ClientManager = manager;
```

Comment:
**Never do this!**

I understand that the hook wants an instance of the ClientManager but:

* In that case it's a strict submodule of the ClientManager and thus instantiated in there
* It should also not be static because wtf
* You can just use a ctor

```csharp
public ClientManager(...)
{
    // ...
    MouseInterceptor = new InterceptMouse(this);
    // ...
}
```